mixer/test/server_test.go

```diff
@@ -83,7 +83,7 @@ func TestCheck(t *testing.T) {
 		t.Fatalf("Failed to connect: %v", err)
 	}
 	defer func() {
-		err = conn.Close()
+		econn.Close()
```

Comment:
does this compile?  where is econn defined?

Reply:
@douglas-reid Yep, that was a typo. I corrected it. Thanks